tests/integration/helpers.bash

```diff
@@ -59,12 +59,10 @@ function __runc() {
 
 # Wrapper for runc spec.
 function runc_spec() {
-	local args=()
-	if [ "$ROOTLESS" -ne 0 ]; then
-		args+=("--rootless")
-	fi
+	local rootless=""
+	[ "$ROOTLESS" -ne 0 ] && rootless="--rootless"
```

Comment:
I got confused here for a bit, and thought that `$ROOTLESS` was a boolean(isn) variable, but it looks to be the `uid` (from a couple of lines up)

```bash
# Check if we're in rootless mode.
ROOTLESS=$(id -u)
```

Do we need the actual `id` in the tests? If not, I'm wondering if we should change that line to set `$ROOTLESS` to a boolean(ish) value (empty or "1"); that would make the comment on it more matching reality, and perhaps make it easier (less error-prone) to use overall

Reply:
I have a follow-up commit which removes ROOTLESS entirely and uses UID instead. The mere aim of this PR is to enable `-u` (which turned out to be quite challenging).

Reply:
https://github.com/opencontainers/runc/pull/3369/commits/dba23d7b24cc21bbcec9d83c96f472f83c1facc7

Reply:
IOW this is addressed in #3369